### ISSUE TITLE
[BUG] Can't delete a record from dashboard listing

### ISSUE THREAD

**Issue by Boorj** [bolt/bolt#5009]
When i go to dashaboard and try to delete something
![2016-03-12 11-17-49](https://cloud.githubusercontent.com/assets/2145313/13721062/5d0769e6-e844-11e5-9918-59a67217cbde.png)

it gives me an error 
![2016-03-12 11-11-01](https://cloud.githubusercontent.com/assets/2145313/13721065/6143bc8a-e844-11e5-9a43-c0feef4df71f.png)

in compressed it's here (at first i changed `.parents()` from original to `.closest()` but it doesn't affect)
![2016-03-12 11-11-33](https://cloud.githubusercontent.com/assets/2145313/13721067/7d734cb8-e844-11e5-8085-3550b1ab3358.png)

that leads me here:
![2016-03-12 11-15-49](https://cloud.githubusercontent.com/assets/2145313/13721069/91809e86-e844-11e5-9baa-e54bbcd6056f.png)

But there's no `id` attribute in `<tr/>`.
![2016-03-12 11-16-11](https://cloud.githubusercontent.com/assets/2145313/13721070/96616674-e844-11e5-8794-e01b12956c63.png)

Probably because of 
![2016-03-12 11-19-59](https://cloud.githubusercontent.com/assets/2145313/13721072/9edeec36-e844-11e5-931c-ef97d36f73ae.png)

For me it looks like a bug.

**Comment by GawainLynch** [comment]
Not really helpful mate. I can only presume that this is for a non-root user, etc.  Can you update this with configuration details. Relevant parts of the `permissions.yml` might help to reproduce this problem :wink:

**Comment by Boorj** [comment]
Standard roles, not working under root. =)

#### Relevant part of `permissons.yml`:

```yml
# - 'root', the "superuser" role; Bolt will automatically grant all permissions
```

#### Irrelevant part of `permissons.yml` (just for a case;) :
```yml
roles:
    editor:
        description: This user can edit some content.
        label: Editor
    chief-editor:
        description: This user can edit any content in the system.
        label: Chief Editor
    admin:
        description: User-level administrator.
        label: Administrator
    developer:
        description: Developer access. Only required to change system-level settings.
        label: Developer
```

What else you may need?

**Comment by Boorj** [comment]
#### Investigation ʢ◉ᴥ◉ʡ
###### Step 1
the problem is in `prop.extended`, not in `modifiable`

**Comment by GawainLynch** [comment]
Right! Got you now, you're correct this is nothing to do with `permissions.yml`.

Escalating and ping @rarila

**Comment by GawainLynch** [comment]
Fixed in #5131
